src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -639,7 +641,6 @@ public void toCopy() throws WorkflowException {
 
         super.toCopy();
 
-        KNSGlobalVariables.getMessageList().clear();
```

Comment:
It wasn't causing the part where stuff was being cleared that shouldn't, but was causing an issue where messages from the clearing method weren't being displayed like they should be.